layers/+lang/markdown/README.org

```diff
@@ -47,6 +50,11 @@ For more extensive support of references with BibTeX files, have a look at the
 To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =markdown= to the existing =dotspacemacs-configuration-layers= list in this
 file.
+** grip-mode
+Grip-mode [[https://github.com/seagle0128/grip-mode#prerequisite][requires python and the python package grip]] to be installed on the
+system. Grip can usually be installed with the command =pip install grip=,
+on debian based systems make sure you use the python 3 version
```

Comment:
No need to include installation instruction for specific distributuon.

Reply:
This is literally copy/paste from the `github` layer.

Reply:
I proposed changes earlier. We may as well rewrite the description while we're moving it from the deprecated layer.

Reply:
Resolved.